docs/overview_and_guide.rst

```diff
@@ -126,6 +126,7 @@ In order to use Raiden correctly and safely there are some things that need to b
 - **Persistency of local DB**: Your local state database is located at ``~/.raiden``. This data should not be deleted by the user or tampered with in any way. Frequent backups are also recommended. Deleting this directory could mean losing funds.
 - **Raiden Always online**: Make sure that your node is always working, your network connection is stable and that the Raiden node is always online. If it crashes for whatever reason you are responsible to restart it and keep it always online. We recommend running it inside some form of monitor that will restart if for some reason the raiden node crashes.
 - **Ethereum Client Always Online**: Make sure that your ethereum client is always running and is synced. We recommend running it inside some form of monitor that will restart if for some reason it crashes.
+- **Ethereum Client can not be changed**: Swapping the ethereum client while transactions are not mined is considered unsafe. We recommend avoiding switching ethereum clients once the Raiden node is running.
```

Comment:
Isn't this also related with the Ethereum client staying always online? I guess you can't really swap the client even if you try at the moment. The moment we lose RPC connection with the client we exit with the following:

```
Could not contact the ethereum node through JSON-RPC.
Please make sure that JSON-RPC is enabled for these interfaces:

    eth_*, net_*, web3_*

geth: https://github.com/ethereum/go-ethereum/wiki/Management-APIs
```

Reply:
It definitely is related to the Ethereum client being online (you can't swap the client without stopping the running one). But IMO this two things are different enough to be worth the extra line.

Reply:
> Well, you could stop Raiden and restart pointing to a different Ethereum node on another port)

But is that bad? Raiden would shut down gracfully and wait till all transactions are mined, right?

Reply:
> Raiden would shut down gracfully

No, we have a requirement just above stating that Raiden should always be running, this is because we don't have graceful shutdown anymore. (Unless you have a different definition of graceful than I do)

> Raiden would shut down gracfully and wait till all transactions are mined, right?

It doesn't

Reply:
I thought we were waiting before transactions were mined, before quitting. If not then disregard this.

>      It doesn't. This cannot be guaranteed under crashes.

That is correct. That's also a good point then. If raiden crashes and we then restart and point to another raiden node it would be bad. Which is what the note you are adding should convey.